src/services/teacher.ts

```diff
@@ -93,4 +95,17 @@ export class TeacherService {
             return this.teacherRepository.delete(deleteTeacher.id)
         }
     }
+
+    public async uploadTeachers(filename: string) {
+        const teachers = [];
+
+        console.log(filename)
+        fs.createReadStream(`./src/uploads/${filename}`)
+            .pipe(csvParser({}))
+            .on('data', (data) => teachers.push(data))
```

Comment:
how are you omitting the first row which contains column headers?

Reply:
it automatically omitting sir its directly taking the data only sir